garden-backend-service/migrations/versions/f293268b31c5_add_fields_to_users_table_create_users_.py

```diff
@@ -0,0 +1,65 @@
+"""add fields to users table, create users_saved_gardens relationship to support saved gardens
+
+Revision ID: f293268b31c5
+Revises: 11833f8cea54
+Create Date: 2024-07-03 18:52:08.873387
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "f293268b31c5"
+down_revision: Union[str, None] = "11833f8cea54"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "users_saved_gardens",
+        sa.Column("user_id", sa.Integer(), nullable=True),
+        sa.Column("garden_id", sa.Integer(), nullable=True),
```

Comment:
I think the change to add the primary key constraint slipped through. IMO nothing wrong with having multiple migration scripts in a single PR in the future so we can keep using --autogenerate instead of doing stuff by hand 
```suggestion
        sa.Column("user_id", sa.Integer(), nullable=False),
        sa.Column("garden_id", sa.Integer(), nullable=False),
```